musician-details.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import Modal from 'react-bootstrap4-modal'
+
+export default class MusicianDetails extends React.Component {
+  render() {
+    const musician = this.props.musician
+    return (
+      <div>
+        <Modal
+          visible={!!musician}
+          dialogClassName="modal-dialog-centered modal-lg"
+          onClickBackdrop={this.props.close}
+        >
+          <div className="modal-header">
+            <button
+              type="button"
+              className="close"
+              aria-label="Close"
+              onClick={this.props.close}
+            >
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div className="modal-body container text-center">
+            {musician ? (
```

Comment:
It's better to use `&&` [conditional rendering](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) instead of a ternary if one of your ternary branches basically does nothing. In any case, `musician` is always "truthy" if it's an `Array`.